omnisharp.json

```diff
@@ -0,0 +1,16 @@
+{
+    "FormattingOptions": {
+        "NewLinesForBracesInLambdaExpressionBody": false,
+        "NewLinesForBracesInAnonymousMethods": false,
+        "NewLinesForBracesInAnonymousTypes": false,
+        "NewLinesForBracesInControlBlocks": false,
+        "NewLinesForBracesInTypes": false,
+        "NewLinesForBracesInMethods": false,
+        "NewLinesForBracesInProperties": false,
+        "NewLinesForBracesInObjectCollectionArrayInitializers": false,
+        "NewLinesForBracesInAccessors": false,
+        "NewLineForElse": false,
+        "NewLineForCatch": false,
+        "NewLineForFinally": false
+    }
+}
```

Comment:
https://github.com/dotnet/vscode-csharp/issues/1506

Reply:
Does this apply to Visual Studio?

Reply:
I believe it's a config file for this extension - https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp.

For built-in formatter, [.editorconfig](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files) file will be needed.

Reply:
I have always used VSCode as my editor, but I think this should be doable. There is also a plugin for VSCode to apply the .editorconfig formatting.